tests/e2e/docs/vanilla_cluster_setup.md

```diff
@@ -22,6 +22,7 @@ The section outlines how to set the env variable for running e2e test.
 
     [Global]
     insecure-flag = "true"
+    cluster-id = "<cluster-id>"
```

Comment:
is this a UUID ? or a mo-ref id ?
Can you confirm

Reply:
it is the cluster name as seen in VC

Reply:
Updated the md file with the same.